tests/libfuzzer/core_helper_fuzzer/libfuzz_harness.cpp

```diff
@@ -5,24 +5,39 @@
 
 #include "fuzz_helper_function.hpp"
 
+struct fuzz_xdp_md_helper_t : public xdp_md_helper_t
+{
+  public:
+    fuzz_xdp_md_helper_t() : xdp_md_helper_t(packet) {}
+
+  private:
+    uint8_t packet_buffer[MAX_BUFFER_SIZE] = {0};
+
+    std::vector<uint8_t> packet{packet_buffer, packet_buffer + sizeof(packet_buffer)};
+};
+
+typedef fuzz_helper_function<fuzz_xdp_md_helper_t> fuzz_helper_function_xdp_t;
+std::unique_ptr<fuzz_helper_function_xdp_t> _fuzz_helper_function_xdp;
+
 typedef fuzz_helper_function<bpf_sock_addr_t> fuzz_helper_function_sock_addr_t;
 std::unique_ptr<fuzz_helper_function_sock_addr_t> _fuzz_helper_function_sock_addr;
 
 typedef fuzz_helper_function<bpf_sock_ops_t> fuzz_helper_function_sock_ops_t;
 std::unique_ptr<fuzz_helper_function_sock_ops_t> _fuzz_helper_function_sock_ops;
 
 int selected_program_type = 0;
-FUZZ_EXPORT int __cdecl LLVMFuzzerInitialize(int* argc, char*** argv)
+FUZZ_EXPORT int __cdecl LLVMFuzzerInitialize(int *argc, char ***argv)
```

Comment:
Revert this line.  The original is the style used in this repo.  And regardless, this change is not part of bringing back the XDP mock provider.